.github/actions/generate-builder/action.yml

```diff
@@ -43,17 +43,6 @@ runs:
     - name: Generate builder
       shell: bash
       env:
-        # Builder.
-        BUILDER_REPOSITORY: slsa-framework/slsa-github-generator  # The repository to download the pre-built builder binary from.
-        BUILDER_RELEASE_BINARY: "${{ inputs.binary }}"            # The name of the pre-built binary in the release assets. This is also used as the final binary name when compiling the builder from source.
-        # Verifier
-        # NOTE: These VERIFIER_* variables are used in the builder-fetch.sh script for verification of builder
-        # release binaries when the compile-builder input is false.
-        VERIFIER_REPOSITORY: slsa-framework/slsa-verifier         # The repository to download the pre-built verifier binary from.
-        VERIFIER_RELEASE_BINARY: slsa-verifier-linux-amd64        # The name of the verifier binary in the release assets.
-        VERIFIER_RELEASE_BINARY_SHA256: f92fc4e571949c796d7709bb3f0814a733124b0155e484fad095b5ca68b4cb21  # The expected hash of the verifier binary.
-        VERIFIER_RELEASE: v1.1.1                                   # The version of the verifier to download.
-
```

Comment:
These environment variables are necessary for supporting when `compile-builder` is false so they should not be deleted.

Reply:
The environment variables being set isn't really a problem. The script that runs as part of that action just changes behavior based on the `compile-builder` flag. However, the container workflow will not work yet with `compile-builder` set to false because we haven't created a release that includes the container workflow.

Currently you will need to use `main` or a recent commit digest in your workflow call and set `compile-builder` to true for it to work.

Once we cut a release that includes the container workflow then setting it to false should work but not until then.

Reply:
Added #613 to track this change.